edit/edit.go

```diff
@@ -290,6 +290,45 @@ func where(ed *Editor, a Address) (addr, error) {
 	return at, nil
 }
 
+// WhereLine returns the line number corresponding to the start and
+// end of the address.
+// Dot is set to the address.
+func (ed *Editor) WhereLine(a Address) (lineFrom, lineTo int64, err error) {
```

Comment:
Addresses use int64, may as well be consistent, so that my file filled with ^int(0) newlines is editable in T.